src/pages/settings/AddSecondaryLoginPage.js

```diff
@@ -125,10 +127,10 @@ class AddSecondaryLoginPage extends Component {
                                     : 'profilePage.emailAddress')}
                             </Text>
                             <TextInput
+                                ref={e => this.phoneNumberInputRef = e}
```

Comment:
Typically, we use `el` for the shorthand on refs. `e` would usually be used for events. Would you mind changing this?